.github/workflows/ci-build.yml

```diff
@@ -23,21 +23,22 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python }}
+      - name: Install poetry
+        run: |
+          pipx install poetry
       - name: Build a binary wheel
         run: |
-          python -m pip install numpy --compile --pre
-          python -m pip install --user -U pip wheel setuptools
-          python setup.py bdist_wheel
+          poetry build
       - name: Install LedFx
         run: |
-          pip install --pre --find-links=dist .
+          poetry install
 
   build-ledfx-windows:
     name: Build LedFx (Windows)
     runs-on: windows-latest
     strategy:
       matrix:
-        python: [3.8,3.9,3.10.x]
```

Comment:
any reason that 3.9 does not have an .x

I don't know the syntax here, but that looks suspicous